database/dbase.go

```diff
@@ -24,8 +24,8 @@ func newDbVersion(version string) (*dbVersion, error) {
 		return nil, fmt.Errorf("version %s does not match the required format", version)
 	}
 	// We don't test the Atoi errors as version matches the regexp
-	var maj, _ = strconv.Atoi(match[0])
-	var min, _ = strconv.Atoi(match[1])
+	var maj, _ = strconv.Atoi(match[1])
```

Comment:
As it was source of an issue maybe we could add a comment to explain why we use 1 and 2. When someone will read it, it may think it is an error.